oracle_test.go

```diff
@@ -89,42 +89,73 @@ func TestOracle_Initialize(t *testing.T) {
 }
 
 func TestOracle_CreateUser(t *testing.T) {
-	connURL, cleanup := prepareOracleTestContainer(t)
-	defer cleanup()
+	type testCase struct {
```

Comment:
This diff got mangled. tl;dr of what I did:
- Wrapped a table test around the existing test
- Moved the empty creations test into a test case which required a few changes to the assertions